lib/wings/attribute_transformer.rb

```diff
@@ -5,8 +5,16 @@
 module Wings
   ##
   # Transform AF attributes to Valkyrie::Resource attributes representation.
-  #
   class AttributeTransformer
+    def self.for(obj)
```

Comment:
we need different behavior for some objects, a factory lets us switch without a lot of branching.